packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -1680,6 +1693,17 @@ export class ProjectService {
 
                     Logger.debug(`Fetch query results from cache or warehouse`);
                     span.setAttribute('generatedSql', query);
+
+                    /**
+                     * If enabled, we include additional attributes for this span allowing us to measure
```

Comment:
do you also want a boolean to indicate the flag explicitly? Rather than having "if tableCalculationsNum is there then we are using new table calculations"?

Reply:
Definitely, was going to add that in a follow-up but there's really no reason not to include it here - `tableCalculationsNum` will stop being an indicator of the feature flag once we're measuring the impact of any changes.